fda/datasets/_real_datasets.py

```diff
@@ -147,7 +147,7 @@ def fetch_ucr(name, **kwargs):
     """
 
 
-def fetch_phoneme(return_X_y: bool = False):
+def fetch_phoneme(return_X_y: bool=False):
```

Comment:
Actually, your editor was right separating the equals when there is a type annotation.

Reply:
cheanged